pallets/subtensor/src/coinbase/root.rs

```diff
@@ -592,6 +593,11 @@ impl<T: Config> Pallet<T> {
     pub fn get_network_to_prune() -> Option<NetUid> {
         let current_block: u64 = Self::get_current_block_as_u64();
 
+        if let Some(priority_netuid) = Self::pop_ready_subnet_deregistration_priority(current_block)
```

Comment:
```suggestion
        if let Some(priority_netuid) = Self::pop_ready_subnet_deregistration_priority()
```

Unnecessary passed arg